demos/tests/cases.py

```diff
@@ -239,25 +246,29 @@ def single_option_cases(key, *args):
     )),
 
     NativeDemo(subdirectory='super_resolution_demo', device_keys=['-d'], test_cases=combine_cases(
-        TestCase(options={'-i': ImageDirectoryArg('single-image-super-resolution')}),
+        TestCase(options={'-i': DataDirectoryArg('single-image-super-resolution')}),
         TestCase(options={
             '-m': ModelArg('single-image-super-resolution-1033'),
         }),
     )),
 
     NativeDemo(subdirectory='text_detection_demo', device_keys=['-d_td', '-d_tr'], test_cases=combine_cases(
         TestCase(options={'-no_show': None, '-dt': 'video',
-            '-i': ImagePatternArg('text-detection')}),
+            '-i': DataPatternArg('text-detection')}),
         single_option_cases('-m_td', ModelArg('text-detection-0003'), ModelArg('text-detection-0004')),
         single_option_cases('-m_tr', None, ModelArg('text-recognition-0012')),
     )),
 ]
 
 PYTHON_DEMOS = [
-    # TODO: 3d_segmentation_demo: no input data
+    PythonDemo(subdirectory='3d_segmentation_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'-m': ModelArg('brain-tumor-segmentation-0001'),
+                          '-o': '.'}),
+        single_option_cases('-i', *DATA_SEQUENCES['brain-tumor-nifti']),
```

Comment:
Given that this sequence is of one item, you can just inline that item here, and then merge the `-i` option into the above `TestCase` call. Same goes for image retrieval demo.

Reply:
I would agree with you, but because we are planning to validate demos, this code is more agile.

Reply:
How so?

Reply:
It's easy to manipulate with data in such manner.

Reply:
I don't really get it, but... sure, if you insist.